src/border_agent/border_agent.hpp

```diff
@@ -154,6 +153,7 @@ class BorderAgent : private NonCopyable
         uint32_t ToUint32(void) const;
     };
```

Comment:
What about moving those classes to .cpp file as well?

Reply:
Done